rclcpp/test/test_logging.cpp

```diff
@@ -86,7 +101,7 @@ TEST_F(TestLoggingMacros, test_logging_named) {
   if (g_last_log_event.location) {
     EXPECT_STREQ("TestBody", g_last_log_event.location->function_name);
     EXPECT_THAT(g_last_log_event.location->file_name, EndsWith("test_logging.cpp"));
-    EXPECT_EQ(82u, g_last_log_event.location->line_number);
+    EXPECT_EQ(97u, g_last_log_event.location->line_number);
```

Comment:
I think that we can avoid updating this number in the future.
From line 96:
```cpp
  for (int i : {1, 2, 3}) {
    RCLCPP_DEBUG(g_logger, "message %d", i);
  }
size_t expected_location = __LINE__ - 2u;
```
Then this line could be:
```suggestion
    EXPECT_EQ(expected_location, g_last_log_event.location->line_number);
```